astropy/io/votable/tree.py

```diff
@@ -3404,9 +3404,9 @@ def __init__(self, ID=None, id=None, config=None, pos=None, version="1.4"):
         self._groups = HomogeneousList(Group)
 
         version = str(version)
-        if version not in ("1.0", "1.1", "1.2", "1.3", "1.4"):
-            raise ValueError("'version' should be one of '1.0', '1.1', "
-                             "'1.2', '1.3', or '1.4'")
+        if (version != '1.0') and (version not in self._version_namespace_map):
```

Comment:
Maybe something like this? Need these imports:

```python
import warnings

from astropy.utils.exceptions import AstropyDeprecationWarning
```

```suggestion
        if version == '1.0':
            warnings.warn('VOTable 1.0 support is deprecated in astropy 4.3 and will be '
                          'removed in a future release', AstropyDeprecationWarning)
        elif (version != '1.0') and (version not in self._version_namespace_map):
```

Reply:
p.s. If you decide to deprecate, need a second change log, maybe `api` instead of `bugfix`.

Reply:
Thanks for this, and the entire review!  

All the code changes have been made and committed.  I should have thought of the change log.  Since the CI looks clean,  I'll add the change log file now with a ` [ci skip]` in the commit.